src/main/java/com/ita/if103java/ims/dao/UserDaoImpl.java

```diff
@@ -0,0 +1,217 @@
+package com.ita.if103java.ims.dao;
```

Comment:
for implementation better to create separate package com.ita.if103java.ims.dao.impl